apps/ui/src/views/Space/Editor.vue

```diff
@@ -146,6 +154,15 @@ const canSubmit = computed(() => {
     ? votingPower.value?.canPropose
     : !web3.value.authLoading;
 });
+const spaceType = computed(() =>
+  props.space.turbo ? 'turbo' : props.space.verified ? 'verified' : 'default'
+);
+
+const proposalLimitReached = computed(
+  () =>
+    (props.space.proposal_count_1d || 0) >= MAX_1D_PROPOSALS[spaceType.value] ||
+    (props.space.proposal_count_30d || 0) >= MAX_30D_PROPOSALS[spaceType.value]
+);
```

Comment:
This will be annoying in testing, is this enforced server-side? I don't think so, do we have plans on enforcing it on testnet? If not would be good to ignore this on testnets.

Reply:
For testing you can change values in https://github.com/snapshot-labs/sx-monorepo/blob/2b3ea26de69ed9b4d874196b652cdf2d43ccda59/apps/ui/src/networks/offchain/api/index.ts#L166-L182 to change proposals counts or values of `verified` or `turbo`

We do enforce this limit on testnet, but all testnet spaces get limits of a verified space 🙈 I think it will be automatically handled with https://github.com/snapshot-labs/workflow/issues/223